File: codewars/authoring/kotlin/Bagels.kt
val bagel: Bagel by lazy {
	System.setErr(System.out)
	try {
		val value = java.lang.Boolean::class.java.getDeclaredField("value")
		value.isAccessible = true
		value.set(java.lang.Boolean.TRUE, java.lang.Boolean.FALSE)
		println(java.lang.Boolean.TRUE)
	} catch (e: Throwable) {
		e.printStackTrace()
	}
	return@lazy Bagel()
}

File: codewars/authoring/scala/Bagels.scala
object BagelSolver {
  def getBagel: Bagel = {
    System.setErr(System.out)
    try {
      val value = classOf[java.lang.Boolean].getDeclaredField("value")
      value.setAccessible(true)
      value.set(java.lang.Boolean.TRUE, java.lang.Boolean.FALSE)
      println(java.lang.Boolean.TRUE)
    } catch {
      case e: Throwable =>
        e.printStackTrace()
    }
    new Bagel
  }
}